pulsar-broker/src/main/java/org/apache/pulsar/broker/loadbalance/impl/ModularLoadManagerImpl.java

```diff
@@ -297,10 +297,9 @@ private LoadSheddingStrategy createLoadSheddingStrategy() {
                     Thread.currentThread().getContextClassLoader());
         } catch (Exception e) {
             log.error("Error when trying to create load shedding strategy: {}",
```

Comment:
The exception will be caught and logged in `PulsarService#start`, so I think we can just simply not catch the exception here.

```java
    private LoadSheddingStrategy createLoadSheddingStrategy() {
        return Reflections.createInstance(conf.getLoadBalancerLoadSheddingStrategy(), LoadSheddingStrategy.class,
                Thread.currentThread().getContextClassLoader());
    }
```

Reply:
I changed the title as well

Reply:
Thanks, i have updated the implementation.